src/core/qgsapplication.cpp

```diff
@@ -878,7 +878,7 @@ void QgsApplication::setUITheme( const QString &themeName )
   {
     // apply OS-specific UI scale factor to stylesheet's em values
     int index = 0;
-    QRegularExpression regex( QStringLiteral( "(?<=[\\s:])([0-9\\.]+)(?=em)" ) );
```

Comment:
This one is safe to make static -- no one should be calling setUiTheme from the non gui thread